### PR TITLE
Add ability to filter registrations in management command

### PR DESCRIPTION
**Pull request by KaitCrawford**
Allows basic filtering of registrations before trying to repopulate their subscriptions

### CODE REVIEW

**Comment by KaitCrawford** [comment]
@smn @miltontony please review

File: registrations/management/commands/repopulate_subscriptions.py [hunk 1]
```diff
@@ -50,7 +55,15 @@ def handle(self, *args, **kwargs):
...
             client = StageBasedMessagingApiClient(sbm_token, sbm_url)
 
-        registrations = Registration.objects.filter(validated=True)
+        filters = {"validated": True}
+        if query:
+            try:
+                query_key, query_value = query.split("=")
+            except ValueError:
+                raise CommandError(
+                    "Please use the format 'key'='value' for --reg-query")
```
Comment (smn) [inline registrations/management/commands/repopulate_subscriptions.py]: Do we perhaps want to use `:` over `=` to differentiate between command line arguments and filtering arguments.

**Comment by KaitCrawford** [comment]
@smn thanks. ready for re-review

File: registrations/management/commands/repopulate_subscriptions.py [hunk 2]
```diff
@@ -50,7 +55,15 @@ def handle(self, *args, **kwargs):
                     'environment variable or --sbm-token is set.')
             client = StageBasedMessagingApiClient(sbm_token, sbm_url)
 
-        registrations = Registration.objects.filter(validated=True)
+        filters = {"validated": True}
+        if query:
+            try:
+                query_key, query_value = query.split(":")
```
Comment (smn) [inline registrations/management/commands/repopulate_subscriptions.py]: sorry missed this earlier but you probably want to use `split(":", 1)` here:

```python
>>> 'foo:bar:baz'.split(':')
['foo', 'bar', 'baz']
>>> 'foo:bar:baz'.split(':', 1)
['foo', 'bar:baz']
```

**Comment by smn** [comment]
:+1:

**Pull request merged**